src/NUnitFramework/framework/Constraints/ComparisonConstraint.cs

```diff
@@ -143,11 +142,43 @@ public ComparisonConstraint Percent
             get
             {
                 _tolerance = _tolerance.Percent;
-                Description += " percent";
                 return this;
             }
         }
 
         #endregion
+        
+        #region Protected Methods
+        
+        /// <summary>
+        /// Provides standard description of what the constraint tests
+        /// based on comparison text.
+        /// </summary>
+        /// <param name="comparisonText">Describes, what constraint tests, throws <see cref="ArgumentNullException"/>
+        /// if null</param>
+        /// <exception cref="ArgumentNullException">Is thrown when null passed to a method</exception>
+        protected string DefaultDescription(string comparisonText)
+        {
+            if (comparisonText == null)
+                throw new ArgumentNullException(nameof(comparisonText), "Comparison text can not be null");
+            
+            System.Text.StringBuilder sb = new System.Text.StringBuilder(comparisonText);
```

Comment:
We would typically add `using System.Text;` it the namespace is used more than once.

Reply:
But it's only used once in `ComparisonConstraint`, from what I see. Thanks for the tip though.

Reply:
I count it as used twice in `System.Text.StringBuilder sb = new System.Text.StringBuilder(...);`.